src/index.html

```diff
@@ -1,13 +1,272 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> -->
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="favicon.svg"
+    >
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <div class="header-links header__navigation">
+        <a href="index.html" class="header-links__logo">
+          <img src="images/logo.png" alt="Bose - brand logo">
+        </a>
+
+        <a href="tel:+123455555555" class="header-links__tel phone">
+          <span class="phone__label">+1 234 5555-55-55</span>
+          <img
+            src="images/phone-icon.svg"
+            alt="Phone icon"
+            class="phone__icon"
+          >
+        </a>
+
+        <a href="#menu" class="header-links__burger burger">
+          <span class="burger__label">Open menu</span>
```

Comment:
If you don't display content of this span you may remove it's text, also consider a different tag maybe a link / button?